repository/sparql_reader.go

```diff
@@ -148,7 +148,19 @@ func (r *SparqlReader) GetAuthorInfo(publication string) (*sparql.Results, error
 				 ?authorship <http://vivoweb.org/ontology/core#relates> ?author .
 				 ?author <http://www.w3.org/2004/02/skos/core#prefLabel> ?author_label .
 			}
-			ORDER BY ?org OFFSET 0 LIMIT 100`, publication)
+			ORDER BY ?author OFFSET 0 LIMIT 100`, publication)
```

Comment:
Can you say more about this change to the order?

Reply:
`?org` was a mistake. There's no org variable in the query.